internal/xds/translator/listener.go

```diff
@@ -30,6 +30,7 @@ import (
 	"google.golang.org/protobuf/types/known/wrapperspb"
 	"k8s.io/utils/ptr"
 
+	egv1a1 "github.com/envoyproxy/gateway/api/v1alpha1"
```

Comment:
we shouldnt be linking any Gateway API fields here, this lib should only be linked with the IR

Reply:
nvm this is internal to this project, but we should ideally avoid this too